### ISSUE TITLE
lts-slim is not working, no module is found

### ISSUE THREAD

**Issue by pzystorm** [ether/etherpad-lite#5316]
**Describe the bug**
When using the latest code from this repo the image lts-slim is used.. But the docker container is not starting and there are a lot of error messages regarding npm modules which are not found.
I have downgraded to 14-slim and use `FROM node:14-slim` in the Dockerfile then everything is working without any issues.

**To Reproduce**
Just git clone and build the container

**Expected behavior**
Etherpad should run

**Screenshots**
```
etherpad    | [2021-12-05 20:44:03.286] [INFO] settings - All relative paths will be interpreted relative to the identified Etherpad base dir: /opt/etherpad-lite
etherpad    | [2021-12-05 20:44:03.294] [INFO] settings - settings loaded from: /opt/etherpad-lite/settings.json
etherpad    | [2021-12-05 20:44:03.294] [INFO] settings - No credentials file found in /opt/etherpad-lite/credentials.json. Ignoring.
etherpad    | [2021-12-05 20:44:03.295] [INFO] settings - Using skin "no-skin" in dir: /opt/etherpad-lite/src/static/skins/no-skin
etherpad    | [2021-12-05 20:44:03.296] [INFO] settings - Session key loaded from: /opt/etherpad-lite/SESSIONKEY.txt
etherpad    | [2021-12-05 20:44:03.296] [INFO] settings - Random string used for versioning assets: 617feaf0
etherpad    | [2021-12-05 20:44:03.555] [INFO] server - Starting Etherpad...
etherpad    | [2021-12-05 20:44:03.657] [INFO] plugins - Running npm to get a list of installed plugins...
etherpad    | [2021-12-05 20:44:03.767] [INFO] plugins - npm --version: 6.14.15
etherpad    | [2021-12-05 20:44:05.784] [ERROR] runCmd|npm - npm WARN read-shrinkwrap This version of npm is compatible with lockfileVersion@1, but package-lock.json was generated for lockfileVersion@2. I'll try to do my best with it!
etherpad    | [2021-12-05 20:44:07.386] [ERROR] runCmd|npm - npm ERR! missing: eslint-config-etherpad@2.0.2, required by etherpad-lite
etherpad    | [2021-12-05 20:44:07.386] [ERROR] runCmd|npm - npm ERR! missing: eslint-plugin-cypress@2.12.1, required by etherpad-lite
etherpad    | [2021-12-05 20:44:07.386] [ERROR] runCmd|npm - npm ERR! missing: eslint-plugin-eslint-comments@3.2.0, required by etherpad-lite
etherpad    | [2021-12-05 20:44:07.387] [ERROR] runCmd|npm - npm ERR! missing: eslint-plugin-mocha@9.0.0, required by etherpad-lite
etherpad    | [2021-12-05 20:44:07.387] [ERROR] runCmd|npm - npm ERR! missing: eslint-plugin-node@11.1.0, required by etherpad-lite
etherpad    | [2021-12-05 20:44:07.387] [ERROR] runCmd|npm - npm ERR! missing: eslint-plugin-prefer-arrow@1.2.3, required by etherpad-lite
etherpad    | [2021-12-05 20:44:07.387] [ERROR] runCmd|npm - npm ERR! missing: eslint-plugin-promise@5.2.0, required by etherpad-lite
etherpad    | [2021-12-05 20:44:07.387] [ERROR] runCmd|npm - npm ERR! missing: eslint-plugin-you-dont-need-lodash-underscore@6.12.0, required by etherpad-lite
etherpad    | [2021-12-05 20:44:07.387] [ERROR] runCmd|npm - npm ERR! missing: etherpad-cli-client@0.1.12, required by etherpad-lite
etherpad    | [2021-12-05 20:44:07.388] [ERROR] runCmd|npm - npm ERR! missing: mocha@9.1.3, required by etherpad-lite
etherpad    | [2021-12-05 20:44:07.388] [ERROR] runCmd|npm - npm ERR! missing: mocha-froth@0.2.10, required by etherpad-lite
etherpad    | [2021-12-05 20:44:07.388] [ERROR] runCmd|npm - npm ERR! missing: nodeify@1.0.1, required by etherpad-lite
etherpad    | [2021-12-05 20:44:07.388] [ERROR] runCmd|npm - npm ERR! missing: openapi-schema-validation@0.4.2, required by etherpad-lite
etherpad    | [2021-12-05 20:44:07.388] [ERROR] runCmd|npm - npm ERR! missing: selenium-webdriver@4.1.0, required by etherpad-lite
etherpad    | [2021-12-05 20:44:07.388] [ERROR] runCmd|npm - npm ERR! missing: set-cookie-parser@2.4.8, required by etherpad-lite
etherpad    | [2021-12-05 20:44:07.388] [ERROR] runCmd|npm - npm ERR! missing: sinon@11.1.2, required by etherpad-lite
etherpad    | [2021-12-05 20:44:07.388] [ERROR] runCmd|npm - npm ERR! missing: split-grid@1.0.11, required by etherpad-lite
etherpad    | [2021-12-05 20:44:07.388] [ERROR] runCmd|npm - npm ERR! missing: supertest@6.1.6, required by etherpad-lite
etherpad    | [2021-12-05 20:44:07.389] [ERROR] runCmd|npm - npm ERR! missing: ep_adminpads2@2.1.46, required by etherpad-lite
etherpad    | [2021-12-05 20:44:07.389] [ERROR] runCmd|npm - npm ERR! missing: ep_mathjax@0.1.12, required by etherpad-lite
etherpad    | [2021-12-05 20:44:07.389] [ERROR] runCmd|npm - npm ERR! missing: ep_tables4@1.0.5, required by etherpad-lite
etherpad    | [2021-12-05 20:44:07.389] [ERROR] runCmd|npm - npm ERR! missing: eslint@7.32.0, required by etherpad-lite
etherpad    | [2021-12-05 20:44:07.389] [ERROR] runCmd|npm - npm ERR! extraneous: ep_etherpad-lite@1.8.16 /opt/etherpad-lite/node_modules/ep_etherpad-lite
etherpad    | [2021-12-05 20:44:07.389] [ERROR] runCmd|npm - npm ERR! extraneous: eslint-config-etherpad@2.0.2 /opt/etherpad-lite/node_modules/eslint-config-etherpad
etherpad    | [2021-12-05 20:44:07.389] [ERROR] runCmd|npm - npm ERR! extraneous: eslint-plugin-cypress@2.12.1 /opt/etherpad-lite/node_modules/eslint-plugin-cypress
etherpad    | [2021-12-05 20:44:07.389] [ERROR] runCmd|npm - npm ERR! extraneous: eslint-plugin-eslint-comments@3.2.0 /opt/etherpad-lite/node_modules/eslint-plugin-eslint-comments
etherpad    | [2021-12-05 20:44:07.389] [ERROR] runCmd|npm - npm ERR! extraneous: eslint-plugin-mocha@9.0.0 /opt/etherpad-lite/node_modules/eslint-plugin-mocha
etherpad    | [2021-12-05 20:44:07.389] [ERROR] runCmd|npm - npm ERR! extraneous: eslint-plugin-node@11.1.0 /opt/etherpad-lite/node_modules/eslint-plugin-node
etherpad    | [2021-12-05 20:44:07.389] [ERROR] runCmd|npm - npm ERR! extraneous: eslint-plugin-prefer-arrow@1.2.3 /opt/etherpad-lite/node_modules/eslint-plugin-prefer-arrow
etherpad    | [2021-12-05 20:44:07.389] [ERROR] runCmd|npm - npm ERR! extraneous: eslint-plugin-promise@5.2.0 /opt/etherpad-lite/node_modules/eslint-plugin-promise
etherpad    | [2021-12-05 20:44:07.389] [ERROR] runCmd|npm - npm ERR! extraneous: eslint-plugin-you-dont-need-lodash-underscore@6.12.0 /opt/etherpad-lite/node_modules/eslint-plugin-you-dont-need-lodash-underscore
etherpad    | [2021-12-05 20:44:07.390] [ERROR] runCmd|npm - npm ERR! extraneous: etherpad-cli-client@0.1.12 /opt/etherpad-lite/node_modules/etherpad-cli-client

...

etherpad    | [2021-12-05 20:44:26.776] [ERROR] runCmd|npm - npm ERR! missing: fast-deep-equal@3.1.3, required by ajv@8.8.2
etherpad    | [2021-12-05 20:44:26.776] [ERROR] runCmd|npm - npm ERR! missing: require-from-string@2.0.2, required by ajv@8.8.2
etherpad    | [2021-12-05 20:44:26.776] [ERROR] runCmd|npm - npm ERR! missing: json-schema-traverse@1.0.0, required by ajv@8.8.2
etherpad    | [2021-12-05 20:44:26.776] [ERROR] runCmd|npm - npm ERR! missing: uri-js@4.4.1, required by ajv@8.8.2
etherpad    | [2021-12-05 20:44:26.794] [ERROR] server - Error occurred while starting Etherpad
etherpad    | [2021-12-05 20:44:26.795] [ERROR] server - Metrics at time of fatal error:
etherpad    | {
etherpad    |   "httpStartTime": 0,
etherpad    |   "memoryUsage": 68837376,
etherpad    |   "memoryUsageHeap": 16358424,
etherpad    |   "ueberdb_lockAwaits": 0,
etherpad    |   "ueberdb_lockAcquires": 0,
etherpad    |   "ueberdb_lockReleases": 0,
etherpad    |   "ueberdb_reads": 0,
etherpad    |   "ueberdb_readsFailed": 0,
etherpad    |   "ueberdb_readsFinished": 0,
etherpad    |   "ueberdb_readsFromCache": 0,
etherpad    |   "ueberdb_readsFromDb": 0,
etherpad    |   "ueberdb_readsFromDbFailed": 0,
etherpad    |   "ueberdb_readsFromDbFinished": 0,
etherpad    |   "ueberdb_writes": 0,
etherpad    |   "ueberdb_writesFailed": 0,
etherpad    |   "ueberdb_writesFinished": 0,
etherpad    |   "ueberdb_writesObsoleted": 0,
etherpad    |   "ueberdb_writesToDb": 0,
etherpad    |   "ueberdb_writesToDbFailed": 0,
etherpad    |   "ueberdb_writesToDbFinished": 0,
etherpad    |   "ueberdb_writesToDbRetried": 0
etherpad    | }
etherpad    | [2021-12-05 20:44:26.795] [ERROR] server - Error: Command exited with code 1: npm ls --long --json --depth=0 --no-production
etherpad    |     at exports (/opt/etherpad-lite/src/node/utils/run_cmd.js:119:25)
etherpad    |     at Object.exports.getPackages (/opt/etherpad-lite/src/static/js/pluginfw/plugins.js:112:48)
etherpad    |     at Object.exports.update (/opt/etherpad-lite/src/static/js/pluginfw/plugins.js:86:34)
etherpad    |     at Object.exports.start (/opt/etherpad-lite/src/node/server.js:143:19)
etherpad    |     at processTicksAndRejections (node:internal/process/task_queues:96:5)
etherpad    | [2021-12-05 20:44:26.795] [INFO] server - Exiting...
etherpad    | [2021-12-05 20:44:26.795] [INFO] server - Waiting for Node.js to exit...
etherpad    | [2021-12-05 20:44:31.795] [ERROR] server - Something that should have been cleaned up during the shutdown hook (such as a timer, worker thread, or open connection) is preventing Node.js from exiting
etherpad    | [2021-12-05 20:44:31.795] [ERROR] server - Enable `dumpOnUncleanExit` setting to get a dump of objects preventing a clean exit
etherpad    | [2021-12-05 20:44:31.795] [ERROR] server - Forcing an unclean exit...
etherpad exited with code 1
```

**Server (please complete the following information):**
 - Etherpad version: 1.8.16
 - OS: Ubuntu 16.04.7 LTS
 - Node.js version (`node --version`): none - only inside the docker
 - npm version (`npm --version`): none - only inside the docker
 - Is the server free of plugins: yes

working version from 14-lts:
```
etherpad@etherpad:/opt/etherpad-lite/var$ node -v
v14.18.2
etherpad@etherpad:/opt/etherpad-lite/var$ npm -v
6.14.15
```

broken version from lts-slim:
```
etherpad@etherpad:/opt/etherpad-lite$ node -v
v16.13.1
etherpad@etherpad:/opt/etherpad-lite$ npm -v
8.1.2
```

**Comment by rhansen** [comment]
> **To Reproduce**
> Steps to reproduce the behavior:
> Just git clone and build the container

I'm unable to reproduce. Please provide detailed steps.

**Comment by MrCyjaneK** [comment]
Same issue, etherpad is failing when running this command:
`npm ls --long --json --depth=0 --no-production`

Here is full output:
```plain
etherpad@oldpc ~ [SIGINT]> cat /home/etherpad/.npm/_logs/2021-12-09T19_29_29_082Z-debug.log
0 verbose cli [
0 verbose cli   '/usr/bin/node',
0 verbose cli   '/usr/local/bin/npm',
0 verbose cli   'ls',
0 verbose cli   '--long',
0 verbose cli   '--json',
0 verbose cli   '--depth=0',
0 verbose cli   '--no-production'
0 verbose cli ]
1 info using npm@7.5.6
2 info using node@v16.13.0
3 timing config:load:defaults Completed in 3ms
4 timing config:load:file:/usr/local/lib/node_modules/npm/npmrc Completed in 4ms
5 timing config:load:builtin Completed in 4ms
6 timing config:load:cli Completed in 5ms
7 timing config:load:env Completed in 1ms
8 timing config:load:file:/home/etherpad/etherpad-lite/.npmrc Completed in 0ms
9 timing config:load:project Completed in 1ms
10 timing config:load:file:/home/etherpad/.npmrc Completed in 1ms
11 timing config:load:user Completed in 1ms
12 timing config:load:file:/etc/npmrc Completed in 0ms
13 timing config:load:global Completed in 0ms
14 timing config:load:cafile Completed in 1ms
15 timing config:load:validate Completed in 0ms
16 timing config:load:setUserAgent Completed in 1ms
17 timing config:load:setEnvs Completed in 2ms
18 timing config:load Completed in 19ms
19 verbose npm-session fc36b95558185c0f
20 timing npm:load Completed in 36ms
21 timing arborist:ctor Completed in 2ms
22 timing command:ls Completed in 1493ms
23 verbose stack Error: extraneous: ep_etherpad-lite@1.8.16 /home/etherpad/etherpad-lite/node_modules/ep_etherpad-lite
23 verbose stack     at ls (/usr/local/lib/node_modules/npm/lib/ls.js:492:7)
23 verbose stack     at processTicksAndRejections (node:internal/process/task_queues:96:5)
24 verbose cwd /home/etherpad/etherpad-lite
25 verbose Linux 5.10.0-7-amd64
26 verbose argv "/usr/bin/node" "/usr/local/bin/npm" "ls" "--long" "--json" "--depth=0" "--no-production"
27 verbose node v16.13.0
28 verbose npm  v7.5.6
29 error code ELSPROBLEMS
30 error extraneous: ep_etherpad-lite@1.8.16 /home/etherpad/etherpad-lite/node_modules/ep_etherpad-lite
31 verbose exit 1
```

**Comment by rhansen** [comment]
@MrCyjaneK Please provide complete steps to reproduce.

**Comment by MrCyjaneK** [comment]
@rhansen so I have no idea - after `git reset --hard` and starting the etherpad it just started to work.. It looks like it was causing by installing plugins using different npm version, if I remember correctly I used this command to install plugins
 - `src/node_modules/npm/bin/npm-cli.js install --no-save --legacy-peer-deps ep_countable`
but I seriously have no idea what happened. It just broke out of nowhere, this instance was untouched for last few months, except for some simple auto update script that executed `git pull`.. `0 5 * * 1 cd etherpad-lite/ && git pull`

TL;DR: for me reinstalling fixed everything

**Comment by tommyhlw** [comment]
just faced the same error with docker image etherpad/etherpad:latest
https://hub.docker.com/layers/etherpad/etherpad/latest/images/sha256-8c3840713cb261fc3628957df28198d65cd993f5adf88f76bab54ff026cd1d01?context=explore
It happen after trying to update https://www.npmjs.com/package/ep_adminpads2

I did reset my volume, except the db-volume and everything works again.
can somebody confirm?
